src/parserindexer/indexer.py

```diff
@@ -43,20 +43,30 @@ def parse_lpsc_from_path(path_str):
         return doc_id, year, doc_url
     return None, None
 
-def map_basic(doc, noalter_prefix="ner"):
+
+# KW: indexer.py isn't handling the 'sentences' field correctly;
+# Solr ingestion fails.  I'm not sure we need it.  
+# So for now, omitting it.
+def map_basic(doc, noalter_prefix=["ner","rel"], nomap_prefix=["sentences"]):
```

Comment:
That's because there is no field in the schema named `sentences`.

Two ways to fix it:
1. add a field named  sentences to schema
2. rename `sentences` to `sentences_ts` and let solr handle it automatically. Note `_ts` means text, multivalued field


Thinking through the code I see `sentences` are retrieved from CoreNLP.
CoreNLP sentences are annotated with POS, NER, lemma etc, So sentences is a list of dictionary objects.
The mlultivalued field in solr handles only simple types like text/string and not complex objects like dictionaries.
Solr will happily accept data if sentences can be converted to list of strings.